moler/connection.py

```diff
@@ -552,3 +552,4 @@ def terminal_thd_conn(name=None):
 _register_builtin_connections()
 if platform.system() == 'Linux':
     _register_builtin_unix_connections()
+    connection_cfg.set_default_variant(io_type="terminal", variant="threaded")
```

Comment:
1. agree - it is good point to set default for builtin connections
2. All the 4 lines should migrate into moler/config/connections.py so that here remains just calls
```suggestion
    connection_cfg.register_builtin_connections(ConnectionFactory, ObservableConnection)
    connection_cfg.set_defaults()
```
Such code I have already at `asyncio_runner` branch (see for reference) - but without that default yet.